docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -22,6 +22,9 @@ Users can configure this limit by setting the provider configuration option `spi
 bin/kc.[sh|bat] --spi-connections-http-client-default-max-consumed-response-size=1000000
 ----
 
+= Password policy for check if password contains Username
```

Comment:
IMO it can be better to add this to the release notes instead of upgrading guide. New password policy does not need to be in upgrading guide as it is new thing and does not affect migration and existing deployments IMO. But release notes (file `25_0_0.adoc` ) can be good.

Feel free to add it right after "argon 2" section as it is related to passwords as well.

Reply:
Moved to 25_0_0.adoc